### PR TITLE
Update readme for 2021.4

### PR DESCRIPTION
**Pull request by pvanderknyff**

### CODE REVIEW

**Comment by lukewrites** [comment]
@pvanderknyff would you mind updating `tdvt/config/tdvt/tdvt.ini` on this PR if you think that makes sense?

**Comment by rosswbrown** [comment]
We should update the prereqs at the bottom of the README as well to reference 2020.4 as the current minimum version. I think this fine to leave the table of all the releases though

**Review by rosswbrown — APPROVED** [review]

File: tdvt/tdvt/config/tdvt/tdvt.ini [hunk 1]
```diff
@@ -1,3 +1,4 @@
 [DEFAULT]
-TAB_CLI_EXE_X64 = C:/Program Files/Tableau/Tableau 2019.2/bin/tabquerytool.exe
-TAB_CLI_EXE_MAC = /Applications/Tableau 2019.2.app/Contents/MacOS/tabquerytool
+# Replace XXXX.X with the version of Tablerau you have installed and wish to test (ie 2021.4)
+TAB_CLI_EXE_X64 = C:/Program Files/Tableau/Tableau XXXX.X/bin/tabquerytool.exe
```
Comment (lukewrites) [inline tdvt/tdvt/config/tdvt/tdvt.ini:3]: I like not having to remember to update this in the future. :shipit:

**Review by lukewrites — APPROVED** [review]
🚢 🇮🇹

**Pull request merged**